README.md

```diff
@@ -138,6 +138,8 @@ This following information is parsed from the integration:
   - V4_CFG_CLUSTER_NODE_POOL_MODE
   - CLUSTER_AUTOSCALER_ACCOUNT
   - CLUSTER_AUTOSCALER_LOCATION
+- Ingress
+  - default value for V4_CFG_INGRESS_MODE (from CLUSTER_API_MODE)
```

Comment:
I think just having `V4_CFG_INGRESS_MODE` would suffice here. Minor change. Like `V4_CFG_POSTGRES_SERVERS` on line 135 above.

It would look like this:

```yaml
- Ingress
    - V4_CFG_INGRESS_MODE (from CLUSTER_API_MODE)
```